lib/src/company/company_service.dart

```diff
@@ -33,10 +33,13 @@ class CompanyService {
       await _enrichService.getCompany(
           domain: domain,
           onSuccess: (company) async {
+
+            if (company?.about?.domain != null) domain = company!.about!.domain!;
```

Comment:
instead of
if (company?.about?.domain != null) domain = company!.about!.domain!;
use
domain = company?.about?.domain != null ? company!.about!.domain! : domain;